calculator-data/src/main/java/camp/nextstep/edu/calculator/data/model/RecordEntity.kt

```diff
@@ -0,0 +1,25 @@
+package camp.nextstep.edu.calculator.data.model
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import camp.nextstep.edu.calculator.domain.model.Record
+import java.util.*
+
+@Entity(tableName = "records")
+data class RecordEntity(
+    @PrimaryKey val id: UUID = UUID.randomUUID(),
```

Comment:
```suggestion
    @PrimaryKey(autoGenerate = true) val id: Long = 0,
```

디비에서 id는 고유 식별자로써 역할을 하게 되는데요.
난수보단 생긴 순서대로 저장해주는 것이 더욱 좋습니다
위와 같이 진행하면 자동으로 저장될 때마다 1개씩 카운트가 올라가며 id값이 저장됩니다~

Reply:
randomUUID는 숫자 자리수 때문에 용량적인 문제로 단점이라 생각하는데 그외에 다른 단점이 있을까요?

Reply:
저는 숫자 자리수의 내용보다는 난수인 것이 문제라고 생각됩니다.
id가 순차적으로 증가하는 것이 포인트라고 생각되는데요
db에서 indexing 작업을 수행할 때 id가 기준이 될 수도 있겠네요~!

Reply:
그렇군요!